signer/src/error.rs

```diff
@@ -237,6 +241,11 @@ pub enum Error {
     #[error("invalid tweak? seriously? {0}")]
     InvalidPublicKeyTweak(#[source] secp256k1::Error),
 
+    /// This happens when a tweak produced by [`XOnlyPublicKey::add_tweak`] was computed incorrectly.
+    /// One if possible reasons is that you tweaked same key twice.
```

Comment:
I would remove the second line, I'm not sure that it would point us in the right direction for the cause of the error. It's not obvious what the error really means here, as in I'm not sure why an error is possible to begin with.
```suggestion
    /// This happens when a tweak produced by [`XOnlyPublicKey::add_tweak`] was computed incorrectly.
```